packages/vue-app/template/store.js

```diff
@@ -3,130 +3,136 @@ import Vuex from 'vuex'
 
 Vue.use(Vuex)
 
-let storeData = {}
-
-let files
+const VUEX_PROPERTIES = ['state', 'getters', 'actions', 'mutations']
+let store = {}
+let fileResolver
```

Comment:
Is there any advantage to have this as global variable? We could pass in `fileResolver(path)` to `requireModule` to avoid that

Reply:
I don't see your point here, it's just to avoid re-creating the array in memory with HMR

Reply:
But won't this happen anyway because of https://github.com/nuxt/nuxt.js/blob/4e0ab55ed8ea57b107fdf2157fb781ba22f790b3/packages/vue-app/template/store.js#L9-L12